frontend/src/components/admin/userManagement/UserAddModify.js

```diff
@@ -763,6 +773,16 @@ function UserAddModify() {
     }
   };
 
+  useEffect(() => {
+    if (!userDataShow.currentPassword || !userDataShow.userPassword) {
+      setIsCurrentPassCorrect(false);
+    } else if (userDataShow.currentPassword !== userDataShow.userPassword) {
+      setIsCurrentPassCorrect(false);
+    } else {
+      setIsCurrentPassCorrect(true);
```

Comment:
you cant correctly check a current user password from the forntend since its harshed.
You can only check it form the backend side ie[ here](https://github.com/DIGI-UW/OpenELIS-Global-2/blob/develop/src/main/java/org/openelisglobal/login/controller/ChangePasswordLoginController.java#L71)